scripts/create_markets_from_sheet_omen.py

```
@@ -0,0 +1,100 @@
+
+
+def main(
+    path: str,
+    category: str = typer.Option(),
+    initial_funds: str = typer.Option(),
+    from_private_key: str = typer.Option(),
+    safe_address: str = typer.Option(None),
+    cl_token: CollateralTokenChoice = CollateralTokenChoice.sdai,
+    fee_perc: float = typer.Option(OMEN_DEFAULT_MARKET_FEE_PERC),
+    language: str = typer.Option("en"),
+    outcomes: list[str] = typer.Option(OMEN_BINARY_MARKET_OUTCOMES),
```

Ruff

Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)